src/components/TopNav/NavMenu.js

```diff
@@ -7,19 +7,44 @@ import { Link } from 'react-router-dom'
 import NavigationMenu from 'material-ui/svg-icons/navigation/menu'
 import IconButton from 'material-ui/IconButton'
 import * as auth from 'services/authNService'
+import { _ } from 'underscore'
 
-export const NavMenu = ({ dispatch }) =>
-  <IconMenu
+const Tickets = () => {
+  var persistedTickets  = JSON.parse(localStorage['persist:ticket'])
```

Comment:
In general, if a component needs data, that component needs to get that data from the Redux store using a mapStateToProps function passed into connect. In this case, that would look like:

    const mapStateToNavMenuProps = (state) => {
        currentId = window.location.pathname.match(/(\d+)/)[1] 
        return {
            ticketIds: Object.keys(state.tickets.map).filter(id => id !== currentId)
        }
    }

which would change the NavMenu signature to ({ dispatch, ticketIds })

We should interact with localStorage directly only as a last resort when there's no possible way to get what we need out of the current Redux state. Debugging becomes much easier when we can rely on the fact that everything about the state of our UI is based on the state of the Redux store.

As an added opportunity, this means that the Ticket() function call in the NavMenu component becomes

    ticketIds.map(id => transformIdToTicketLink(id))

or even

    ticketIds.map(id =>
        <MenuItem key={'ticket-' + id} primaryText={<Link to={'/tickets/' + id}>Ticket {id}</Link>}/>
    )